modules/github_stats.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+"""
+github_stats.py - jenni Image Fetcher Module
```

Comment:
Woop, good catch. I copied this from a previous module I wrote and didn't update the name. Will fix and merge down ASAP.